gui/qt/address_list.py

```diff
@@ -81,27 +81,26 @@ def restore_expanded_items(seq_item, used_item, expanded_item_names):
         current_address = item.data(0, Qt.UserRole) if item else None
         expanded_item_names = remember_expanded_items()
         self.clear()
-        receiving_addresses = self.wallet.get_receiving_addresses()
-        change_addresses = self.wallet.get_change_addresses()
 
         if self.parent.fx and self.parent.fx.get_fiat_address_config():
             fx = self.parent.fx
         else:
             fx = None
         account_item = self
-        sequences = [0,1] if change_addresses else [0]
-        for is_change in sequences:
-            if len(sequences) > 1:
-                name = _("Receiving") if not is_change else _("Change")
-                seq_item = QTreeWidgetItem( [ name, '', '', '', '', ''] )
-                account_item.addChild(seq_item)
-                if not is_change and not had_item_count: # first time we create this widget, auto-expand the default address list
-                    seq_item.setExpanded(True)
-            else:
-                seq_item = account_item
+
+        for addr_list, name in [
+            (self.wallet.get_receiving_addresses(), _("Receiving")),
+            (self.wallet.get_change_addresses(), _("Change")),
+            (self.wallet.get_forfeit_addresses(), _("Forfeit"))]:
```

Comment:
For readability, and since the iterating list is never modified, I'd suggest using a tuple and defining it before the loop:

```
addresses_to_name_pairs = (
    (self.wallet.get_receiving_addresses(), _("Receiving")),
    (self.wallet.get_change_addresses(), _("Change")),
    (self.wallet.get_forfeit_addresses(), _("Forfeit")),
)
for addr_list, name in addresses_to_name_pairs:
```